susemanager-utils/susemanager-sls/salt/channels/gpg-keys.sls

```diff
@@ -12,6 +12,16 @@ mgr_deploy_customer_gpg_key:
     - source: salt://gpg/mgr-gpg-pub.key
     - makedirs: True
     - mode: 644
+
+{%- if grains['os_family'] == 'Suse' %}
+mgr_trust_customer_gpg_key:
+  cmd.run:
+    - name: rpm --import /etc/pki/rpm-gpg/mgr-gpg-pub.key
+    - runas: root
+    - require:
+      - file: mgr_deploy_customer_gpg_key
```

Comment:
This will run `rpm --import` every time this state is applied, is that needed? If not, I'd add a [`onchanges`](https://docs.saltproject.io/en/latest/ref/states/requisites.html#onchanges) requisite.

Reply:
I changed this in the PR of 4.3.
As we changed a lot more in 4.3, I will close this PR and create a new one with all the changes we did there